modules/va_facilities/README.v1.yml

```diff
@@ -726,11 +726,22 @@ paths:
     get:
       tags:
         - facilities
-      summary: Retrieve a specific facility by ID
+      summary: Retrieve polygons of the drive time bands for a facility by ID
       description: |
         *This is a draft specification and not an available endpoint.*
+        
+        Retrieve the polygons used to determine in which drive time band veterans fall into for a specified facility. 
+
+        Alternatively, one can retrieve drive time bands for all facilities using `/facilities/all/drive_time_bands`.
+
+        The `ids` parameter is *required* to query this API. Requests without one or the other will return `400 Bad Request`.
```

Comment:
The latest commit says it removes the ids query from the spec, but I'm not seeing that reflected here. Is that correct?

Reply:
I don't know how I managed to not remove this route.